src/providers/bedrock/index.ts

```diff
@@ -56,108 +40,89 @@ import {
 const BedrockConfig: ProviderConfigs = {
   api: BedrockAPIConfig,
   getConfig: (params: Params) => {
-    if (!params.model) {
```

Comment:
Can you please check why this code was removed?

Reply:
Adding it back, (lost some changes in a messy rebase)